docs/reference/filebeat/filebeat-input-journald.md

```diff
@@ -87,10 +87,36 @@ filebeat.inputs:
 
 ### `paths` [filebeat-input-journald-paths]
 
-A list of paths that will be crawled and fetched. Each path can be a directory path (to collect events from all journals in a directory), or a file path. If you specify a directory, Filebeat merges all journals under the directory into a single journal and reads them.
+A list of paths that will be crawled and fetched. Each path can be a
+directory path (to collect events from all journals in a directory),
+or a file path. If you specify a directory, Filebeat merges all
+journals under the directory into a single journal and reads them.
 
 If no paths are specified, Filebeat reads from the default journal.
 
+```yaml
+  - type: journald
+    id: journald-id
+    paths:
+      - /var/log/journal
+```
+
+Will ingest all journals and correctly handle the journald rotation.
```

Comment:
I generally introduce the example and explain in plain language what it does ahead of the code snippet. Somethin glike this:

````suggestion
For example, this configuration will ingest all journals and correctly handle the journald rotation:

```yaml
  - type: journald
    id: journald-id
    paths:
      - /var/log/journal
```
````